src/OpenCensus.Exporter.ApplicationInsights/Implementation/TraceExporterHandler.cs

```diff
@@ -35,235 +36,238 @@ public TraceExporterHandler(TelemetryConfiguration telemetryConfiguration)
             this.telemetryClient = new TelemetryClient(telemetryConfiguration);
         }
 
-        public void Export(IEnumerable<ISpanData> spanDataList)
+        public async Task Export(IEnumerable<ISpanData> spanDataList)
```

Comment:
rename to `ExportAsync` please

Reply:
Done everywhere except unrelated comments/strings.